crengine/include/crsetup.h

```diff
@@ -40,7 +40,6 @@
 #define USE_LIBJPEG                          0
 #define USE_LIBPNG                           0
 #define USE_GIF                              0
-#define USE_FREETYPE                         0
```

Comment:
This was needed, as otherwise `USE_FREETYPE=0` would still qualify as `defined(USE_FREETYPE)`